cvpy/tests/test_cvat_project.py

```diff
@@ -17,12 +17,28 @@
 
 class TestCVATProject(unittest.TestCase):
 
+    def setUp(self) -> None:
+        self.cas_connection = swat.CAS(hostname=TestCVATProject.cas_host, port=TestCVATProject.cas_port,
+                                       username=TestCVATProject.cas_username, password=TestCVATProject.cas_password,
+                                       protocol=TestCVATProject.cas_protocol)
+
+        self.cas_connection.loadactionset('image')
+
+        self.caslib_name = 'dlib'
+
+        self.cas_connection.addcaslib(name=self.caslib_name,
+                                      activeOnAdd=False,
+                                      path=TestCVATProject.datapath,
+                                      dataSource='PATH',
+                                      subdirectories=True)
+
+    def tearDown(self) -> None:
+        self.cas_connection.close()
+
```

Comment:
Thanks for adding these setup/teardown routines! A lot of good cleanup happening below as a result.

Reply:
Ack